wicketd/src/update_tracker.rs

```diff
@@ -1522,6 +1525,7 @@ impl UpdateContext {
                                 StepProgress::with_current_and_total(
                                     progress.current as u64,
                                     progress.total as u64,
+                                    ProgressUnits::BYTES,
```

Comment:
This one is not bytes; the units depend on the component being updated and are kinda weird to specify even then. For the host boot flash the units are something like "sectors erased"; for the sidecar SP it's "aux flash slots scanned for a match". I'm tempted to propose making the units optional?